airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/spec.json

```diff
@@ -116,23 +116,105 @@
       },
       "report_options": {
```

Comment:
Should we delete `report_options` from spec as config migration was added and `report_options` is no longer used?

Reply:
You're right, will delete it now, thanks!